Sources/QuaternionModule/Quaternion.swift

```diff
@@ -32,6 +32,10 @@ import RealModule
 /// Using the infinity norm avoids this problem entirely without significant
 /// downsides. You can access the Euclidean norm using the `length` property.
 /// See `Complex` type of the swift-numerics package for additional details.
+///
+/// `==` does not compare rotations in *R³*; it performs a componentwise and
+/// sign sensitive comparison. You can compare the rotations in *R³* of any
+/// two quaternions using `rotationEquals()`
```

Comment:
Let's try to make this a little bit less technical. How about something like:

> Quaternions are frequently used to represent 3D transformations. It's important to be aware that, when used this way, any quaternion and its negation represent the same transformation, but they do not compare equal using `==` because they are not the same quaternion. You can compare quaternions as 3D transformations using `transformEquals()`.

(Note, "transformEquals" because these aren't pure rotations. Does that seem ok to you?) I think we should also add a "canonicalize transform" operation that selects the representative with sign bit of `x` clear, maps all zeros to (+0, +0, +0, +0), and maps all infinites to a canonical representative (r = .infinity, (x,y,z) zero?).

Reply:
Thats definitely a nicer documentation on the rotation/transform equality behavior, thank you very much!
I do agree that `transformEquals` is probably a better suited method name for a generic quaternion - I'll change it.
Does a property called `canonicalized` – like on `Complex` – seem ok to you for the "canonicalize transform" operation or is it misleading since it (might) also flip the sign of `real`?